src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { getDataURL } = require('./convertToCase/getDataURL');
+const { getErrors } = require('./convertToCase/getErrors');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
+    res.statusMessage = 'OK';
```

Comment:
if i'm not mistaken, you can send just statusCode 200, the 'OK' message will be added automatically

Reply:
agree)